adhoc_reports/parseData.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+wget --no-check-certificate 'https://docs.google.com/spreadsheets/d/1UHyMs5Havel4iEBMPVx7ZY4HMPuhRilIw59a5g733p8/export?gid=1795037732&format=csv' -O loans.csv
+wget --no-check-certificate 'https://docs.google.com/spreadsheets/d/1UHyMs5Havel4iEBMPVx7ZY4HMPuhRilIw59a5g733p8/export?gid=1851646659&format=csv' -O collections.csv
+wget --no-check-certificate 'https://docs.google.com/spreadsheets/d/1UHyMs5Havel4iEBMPVx7ZY4HMPuhRilIw59a5g733p8/export?gid=638160785&format=csv' -O person.csv
+wget --no-check-certificate 'https://docs.google.com/spreadsheets/d/1UHyMs5Havel4iEBMPVx7ZY4HMPuhRilIw59a5g733p8/export?gid=449217660&format=csv' -O connections.csv
```

Comment:
Exposed credentials. depending on the settings in google, anyone with these urls can access those files.

Reply:
I completely forgot about the public nature of this. Can you delete this branch? Then I can make a more publicly neutral version.